docs/gradient-checkpointing.md

```diff
@@ -354,18 +354,24 @@ print_saved_residuals(loss_checkpoint2, params, x, y)
 
 Another policy which refers to names is `jax.checkpoint_policies.save_only_these_names`.
 
-Some of the policies are:
-
-* `everything_saveable`: The default strategy, as if {func}`jax.checkpoint` were not being used at all).
-* `nothing_saveable`: That is, rematerialize everything, as if a custom policy were not being used at all.
-* `dots_saveable`: Or its alias `checkpoint_dots`.
-* `dots_with_no_batch_dims_saveable`: Or its alias `checkpoint_dots_with_no_batch_dims`.
-* `save_anything_but_these_names`: Save any values except for the output of
-  `checkpoint_name` with any of the names given.
-* `save_any_names_but_these`: Save only named values (that is, any outputs of).
-  `checkpoint_name`: Except for those with the names given.
-* `save_only_these_names`: Save only named values, and only among the names
-  given.
+#### List of policies
+
+The policies are:
+* `everything_saveable` (the default strategy, as if `jax.checkpoint` were not being used at all)
+* `nothing_saveable` (i.e. rematerialize everything, as if a custom policy were not being used at all)
+* `dots_saveable` or its alias `checkpoint_dots`
+* `dots_with_no_batch_dims_saveable` or its alias `checkpoint_dots_with_no_batch_dims`
+* `save_anything_but_these_names` (save any values except for the output of
+  `checkpoint_name` with any of the names given)
+* `save_any_names_but_these` (save only named values, i.e. any outputs of
+  `checkpoint_name`, except for those with the names given)
+* `save_only_these_names` (save only named values, and only among the names
+  given)
+* `offload_dot_with_no_batch_dims` same as `dots_with_no_batch_dims_saveable`,
+  but offload to CPU memory instead of recomputing.
+* `save_and_offload_only_these_names` same as `save_only_these_names`, but
+  offload to CPU memory instead of recomputing.
+* `save_from_both_policies(policy_1, policy_2)` This `or` two policies.
```

Comment:
```suggestion
* `save_from_both_policies(policy_1, policy_2)` (like a logical `or`, so that a residual is saveable if it is saveable according to `policy_1` _or_ `policy_2`)
```

Reply:
Actually I think we should rename `save_from_both_policies` to `save_from_either_policy` because to me the name "both" is ambiguous as to whether it means "or" or "and". But we can do that later.